app/models/ingredient.rb

```diff
@@ -99,6 +99,10 @@ def format_name
     self.name = name.downcase
   end
 
+  def measurement_and_name
```

Comment:
Added this method to put the logic in one place. Not sure if you like that or not.

Reply:
I do be liking it.